### DISCUSSION TITLE
Connect InkBox on Kobo Touch to Calibre

### DISCUSSION THREAD

**Discussion by sagresbier** [Quill-OS/quill#75] [Q&A]
Hello Dear,
I've installed InkBox on my Kobo touch. It works like a charm. But I tried to connect to Calibre on my linux machine, but it seems not to work. Could you help me?
Kind regards
sagresbier

**Comment by Szybet** [comment]
It's not supported
Fell free to add support to it, if yes, join or community on Matrix & Discord
This is in my long TODO list somewhere

**Reply by sagresbier** [reply]
Thanks. And to Adobe Digital Editions?

**Reply by Szybet** [reply]
It's proprietary for 90% so no
